File: python/armstrong_numbers.py
def find_armstrong_numbers(numbers):

    answer = []

    for i in numbers:
        if i >= 0 and i < 10:
            i_str = str(i)
            if i == pow(int(i_str[0]), 1) :
                answer.append(i)
        if i >= 10 and i < 100:
            i_str = str(i)
            if i == pow(int(i_str[0]), 2) + pow(int(i_str[1]), 2):
                answer.append(i)
        if i >= 100 and i < 1000:
            i_str = str(i)
            if i == pow(int(i_str[0]), 3) + pow(int(i_str[1]), 3) + pow(int(i_str[2]), 3):
                answer.append(i)

    return answer

File: js/armstrongNumbers.js

exports.findArmstrongNumbers = function(arr) {
    
    let answer = []
    
    for(let i of arr){
        if(i < 10){
            if(i === (Math.pow(i.toString()[0], 1))){
                answer.push(i)
            }
        }
        if(i < 100){
            if(i === (Math.pow(i.toString()[0], 2) + Math.pow(i.toString()[1], 2))){
                answer.push(i)
            }
        }
        if(i < 1000){
            if(i === (Math.pow(i.toString()[0], 3) + Math.pow(i.toString()[1], 3) + Math.pow(i.toString()[2], 3))){
                answer.push(i)
            }
        }
    }
    return answer
}
